index.html

```diff
@@ -10,6 +10,12 @@
         <div class="contaniner__inner-box">
           <div class="row grid">
             {% if site.posts.size > 0 %}
+
+              {% assign frontline_posts = site.pages | where: 'frontline', true %}
```

Comment:
io lo sposterei fuori da `{% if site.posts.size > 0 %}` e lo metterei in un blocco 
`{% if frontline_posts.size > 0 %}`